#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_hashbang_fixes.dpatch by Henrique de Moraes Holschuh <hmh@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix missing, weird or otherwise broken shebang lines

@DPATCH@
diff -urNad hplip~/__init__.py hplip/__init__.py
--- hplip~/__init__.py	2007-01-26 12:24:33.000000000 -0200
+++ hplip/__init__.py	2007-01-26 15:21:09.158181349 -0200
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 #
 # (c) Copyright 2003-2007 Hewlett-Packard Development Company, L.P.
